src/client/styles/components/FilterPopup.scss

```diff
@@ -111,3 +110,17 @@
     z-index: 1000 !important;
   }
 }
+
+.nypl-generic-checkbox > li {
+  margin-left: 0;
+}
+
+.nypl-generic-columns {
+  -webkit-columns: 118px 2;
+  -moz-columns: 118px 2;
+  columns: 118px 2;
+}
+
+.nypl-name-field input[type=text] {
+  -webkit-appearance: none;
```

Comment:
This is for preventing iOS to rewrite the styles for the fields. It caused the border missing.

Reply:
nice one! 🎉